lib/pthreads/README.md

```diff
@@ -11,6 +11,8 @@ This group also provides two optional modules with thread-safe collections:
 
 Theses services are implemented using the POSIX threads.
 
+You can also use the "is threaded" annotation on functions, which makes them run on their own thread. For now, methods with or without parameters are supported, methods with return value are not supported, and methods with self calls are not supported.
```

Comment:
I have some note here:
* We prefere backquote for code blocks: ```  `is threaded` ```.
* In the 2nd line, I'd remove the "methods with or without parameters are supported, " part, it does not convey any useful information (it already works as expected). Keep only the unsupported part.
* Replace "function" with "method", function implies that it returns something, which is an unsupported case.